test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -126,6 +128,8 @@ def scan_pageserver_log_for_errors(
     ".*startup_reconcile: Could not scan node.*",
     # Tests run in dev mode
     ".*Starting in dev mode.*",
+    ".*Starting in dev mode - authentication security checks are disabled.*",
+    ".*Starting in dev mode: this may be an insecure configuration.*",
```

Comment:
Remove all changes in DEFAULT_STORAGE_CONTROLLER_ALLOWED_ERRORS, because you didn't change storage controller's code